bin/wiremock-hearings-tab/wiremock-change-reasons.sh

```diff
@@ -30,7 +30,7 @@ curl -X POST \
     "list_of_values": [
         {
             "category_key": "ChangeReasons",
-            "key": "Differenthearingmoderequired",
```

Comment:
@Marcellofabbri I have changed the keys of the change-reasons based on the values that Simon provided me, are you ok with them?